include/exec/inline_scheduler.hpp

```diff
@@ -54,7 +54,7 @@ namespace exec {
         }
       };
 
-      friend __attrs tag_invoke(stdexec::get_attrs_t, const __sender&) noexcept {
+      friend __attrs tag_invoke(stdexec::get_env_t, const __sender&) noexcept {
```

Comment:
This is picky and unimportant, but the type being returned here should no longer be called `__attrs`. I don't care if you change it though.

Reply:
Thanks for the reminder. There are a number of outstanding types that retain the 'attrs' name that I haven't renamed yet. I'll fix this once I get the more "challenging" open items.